docs/wrapping-react/more-wrapping-examples.md

```diff
@@ -0,0 +1,430 @@
+
+Line `4` defines a functional React component, which on line `8` returns the `MapContainer` which is done in the Reflex code using the `map_container` component.
+
+The `MapContainer` component has props `center`, `zoom`, `scrollWheelZoom`, which we wrap in the `MapContainer` component in the Reflex code. We ignore the `style` prop as it is a reserved name in Reflex. We can use the `rename_props` method to change the name of the prop, as we will see in the React PDF Renderer example, but in this case we just ignore it and add the `width` and `height` props as css in Reflex.
+
+The `TileLayer` component has a prop `url` which we wrap in the `TileLayer` component in the Reflex code.
+
+Lines `24` and `25` defines and exports a React functional component named `Home` which returns the `MapComponent` component. This can be ignored in the Reflex code when wrapping the component as we return the `map_container` component in the `index` function.
+
+
+```javascript
+1 | import dynamic from "next/dynamic";
+2 | import "leaflet/dist/leaflet.css";
+3 | 
+4 | const MapComponent = dynamic(
+5 |   () => {
+6 |     return import("react-leaflet").then((\{ MapContainer, TileLayer }) => {
+7 |       return () => (
+8 |         <MapContainer
+9 |           center=\{[51.505, -0.09]}
+10|           zoom=\{13}
+11|           scrollWheelZoom=\{true}
+12|           style=\{\{ height: "50vh", width: "100%" }}
+13|        >
+14|          <TileLayer
+15|            url="https://\{s}.tile.openstreetmap.org/\{z}/\{x}/\{y}.png"
+16|          />
+17|        </MapContainer>
+18|      );
+19|    });
+20|  },
+21|  \{ ssr: false }
+22| );
+23|
+24| export default function Home() {
+25|   return <MapComponent />;
+26| }
+```
+
+
+```python 
+from reflex.components.component import NoSSRComponent
+from reflex import Var
+
+class MapContainer(NoSSRComponent):
+
+    library = "react-leaflet"
+
+    tag = "MapContainer"
+
+    center: Var[list]
+
+    zoom: Var[int]
+
+    scroll_wheel_zoom: Var[bool]
+
+    # Can also pass a url like: https://unpkg.com/leaflet/dist/leaflet.css 
+    def add_imports(self):
+        return \{"": ["leaflet/dist/leaflet.css"]}
+
+
+
+class TileLayer(NoSSRComponent):
+
+    library = "react-leaflet"
+
+    tag = "TileLayer"
+
+    url: Var[str]
+
+
+map_container = MapContainer.create
+tile_layer = TileLayer.create
+
+def index() -> rx.Component:
+    return map_container(
+                tile_layer(url="https://\{s}.tile.openstreetmap.org/\{z}/\{x}/\{y}.png"),
+                center=[51.505, -0.09], 
+                zoom=13,
+                #scroll_wheel_zoom=True
+                width="100%",
+                height="50vh",
+            )
+
+
+app = rx.App()
+app.add_page(index)
+
+```
+
+
+
+## React PDF Renderer
+
+In this example we are wrapping the React renderer for creating PDF files on the browser and server from this [NPM package](https://www.npmjs.com/package/@react-pdf/renderer).
```

Comment:
```suggestion
In this example we are wrapping the React renderer for creating PDF files on the browser and server from [@react-pdf/renderer](https://www.npmjs.com/package/@react-pdf/renderer).
```

i would suggest naming the package in the links instead of having the link text say "NPM package"

Reply:
changed all 3 of these